js/script.js

```diff
@@ -0,0 +1,145 @@
+$(document).ready(function() {
+
+  // FORM SUBMIT EVENT LISTENER
+  $('form').on('submit', formSubmission)
+
+});
+
+function formSubmission(e) {
+  e.preventDefault();
+
+  var searchType = $('input[name=searchType]:checked', '#myForm').val()
+  var query = $(this).find('#text-field').val();
+  $(this).find('#text-field').val('');
+
+  sendRequest(query, searchType);
+
+}
+
+function sendRequest(query, searchType) {
+  var options = {
+    url: 'https://api.spotify.com/v1/search',
+    data: {
+      q : query,
+      type : searchType,
+      client_id: '1ca2cb5007ff4d5cb5ba9106d6e6929c',
+      client_secret: '20427a5d7f56411ea465b9cc85a39858'
+    }
+  }
+
+  var request = $.ajax(options);
+  request.done(function(response){
+    console.log(response);
+
+    holder = $('.RESULTS');
```

Comment:
make sure to declare your variables. `var holder = `
Also, this is a good instance where you would want to use the `var $holder = ` convention.